app/components/post_header/post_header.js

```diff
@@ -241,7 +253,8 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
             fontSize: 13,
             marginLeft: 5,
             marginBottom: 1,
-            opacity: 0.5
+            opacity: 0.5,
+            marginRight: 5
```

Comment:
maybe you don't need this if you add the margin to the flagContainer style below

Reply:
@enahum I was mimicking what was done with `displayName` and `time` where their combined 10dp spacing came from 5dp each. I could put the whole 10dp on the `flagContainer` though. Thoughts?

Reply:
yes I think is best to do it in the `flagContainer` the other two are always shown and the *flag* is only shown if the post has been flagged

Reply:
@enahum Cool, I'll put the 10dp on the `flagContainer`.